Consolidate home content to custom.mako

ethan-nelson opened the pull request:
Considering people who host other installations of the TM as well as parts of the issues raised in #382, #720, #410, and a few others, I think it would be better to facilitate content on the home page by moving it all into the `custom.mako` file.

By doing so, this will hopefully provide users interested in installing a custom instance with non-HOT branding a one-stop-shop file to customize text throughout the TM--namely `custom.mako`. A user can already change the instance name, footer information, and some parts of the front page in this file, so I don't think it's too much of a stretch to just populate the full right half of the front page from it. 

In addition, I have consolidated the same text from `/` and `/about` into one variable and use it among both pages to ensure they match.

Any thoughts?

pgiraud commented:
Hi @ethan-nelson
This is a nice initiative.
There's only one thing that bothers me. You seem to have added a "<br/>" in the about intro. Though I agree with you that it looks better with more space, it's usually a bad idea to change the source msg, otherwise we loose the translation.
An other option is to update the translations manually. I can give a hand on that. I already did it once.
What do you think?

Moreover, in the given example, I would even prefer using "paragraphs" instead of 2 row breaks.

ethan-nelson commented:
Alright, since we'll have to regenerate translations I will go ahead and transition this to paragraphs.

ethan-nelson commented:
I think if I understood correctly you were okay with breaking the translations and regenerating them if we're going to completely reshuffle these things around. (If that's not the case, then I can reset the three commits I just added). In three commits, I:

1. Broke up all `<br />` into paragraph chunks
2. Added links to MapGive and the LearnOSM TM tutorial (see #685, #627, #382)
3. Some minor copyediting.

So I rolled up a few different things in here, but what is actually desired may be up to just commit # 1.

pgiraud commented:
It looks good. I'll merge it myself as soon as possible and will make the adequate manual changes in the translation files.

ethan-nelson commented:
Great! Thanks for doing that.
